scipy/ndimage/fourier.py

```diff
@@ -111,6 +111,24 @@ def fourier_gaussian(input, sigma, n=-1, axis=-1, output=None):
         The filtered input. If `output` is given as a parameter, None is
         returned.
 
+    Examples
+    --------
+    >>> from scipy import ndimage, misc
+    >>> import numpy.fft
+    >>> import matplotlib.pyplot as plt
+    >>> fig = plt.figure()
+    >>> plt.gray()  # show the filtered result in grayscale
+    >>> ax1 = fig.add_subplot(131)  # left side
+    >>> ax2 = fig.add_subplot(132)  # middle
+    >>> ax3 = fig.add_subplot(133)  # right side
+    >>> ascent = misc.ascent()
+    >>> input_ = numpy.fft.fft2(ascent)
+    >>> result = ndimage.fourier_gaussian(input_, sigma=4)
+    >>> result = numpy.fft.ifft2(result)
+    >>> ax1.imshow(ascent)
+    >>> ax2.imshow(result.real)
+    >>> ax3.imshow(result.imag)
```

Comment:
For the last commit, there might be some examples where color would also make sense. For this, I think it makes it just simpler, as I would have to apply the functions to each channel otherwise (see http://stackoverflow.com/a/9827229/562769 )